src/vst3/SurgeVst3Processor.cpp

```diff
@@ -1007,12 +1007,13 @@ void SurgeVst3Processor::handleZoom(SurgeGUIEditor *e)
             Steinberg::tresult res = ipf->resizeView(e, &vr);
             if (res != Steinberg::kResultTrue)
             {
-               std::ostringstream oss;
+			   // Leaving this here for debug purposes. resizeView() can fail in non-fatal way and zoom reset is just too harsh.
+               /*std::ostringstream oss;
                oss << "Your host failed to zoom VST3 to " << e->getZoomFactor() << " scale. "
                    << "Surge will now attempt to reset the zoom level to 100%. You may see several "
                    << "other error messages in the course of this being resolved.";
                Surge::UserInteractions::promptError(oss.str(), "VST3 Host Zoom Error" );
-               e->setZoomFactor(100);
+               e->setZoomFactor(100);*/
```

Comment:
Not sure if this actually fixes a problem in some host. In it's not necessary at all and in fact breaks resize when playing with the zoom menu.

Reply:
That code is there to fix Studio One. https://github.com/surge-synthesizer/surge/issues/2397

We will need to retest this change carefully in that DAW

Reply:
Maybe this is the part where daw == cubase detection is needed then. I can add it.

Reply:
lets see what @mkruselj sees when he tests it first. We could also put an “if studio one” just as easily.

Reply:
S1 seems good without it now!